grub-core/kern/i386/pc/tpm.c

```diff
@@ -0,0 +1,132 @@
+#include <grub/err.h>
+#include <grub/i18n.h>
+#include <grub/mm.h>
+#include <grub/tpm.h>
+#include <grub/misc.h>
+#include <grub/i386/pc/int.h>
+
+int tpm_present (void);
```

Comment:
inconsistent formatting with the rest of the function signatures, especially 2 lines down, why all the spaces?